tools/monorepo-utils/src/changefile/lib/projects.ts

```diff
@@ -72,27 +77,32 @@ export const getChangeloggerProjectPaths = async (
  * @param {string} base        base hash
  * @param {string} head        head hash
  * @param {string} fileName    changelog file name
+ * @param {string} baseOwner   PR base owner
+ * @param {string} baseName    PR base name
  * @return {Array<string>} List of files changed in a PR.
  */
 export const getTouchedFilePaths = async (
 	tmpRepoPath: string,
 	base: string,
 	head: string,
-	fileName: string
+	fileName: string,
+	baseOwner: string,
+	baseName: string
 ) => {
 	const git = simpleGit( {
 		baseDir: tmpRepoPath,
 		config: [ 'core.hooksPath=/dev/null' ],
 	} );
 
-	// make sure base sha is available.
+	//make sure base sha is available.
 	await git.raw( [
 		'remote',
 		'add',
-		'woocommerce',
-		'git@github.com:woocommerce/woocommerce.git',
+		baseOwner,
+		getAuthenticatedRemote( { owner: baseOwner, name: baseName } ),
 	] );
-	await git.raw( [ 'fetch', 'woocommerce', base ] );
+	await git.raw( [ 'fetch', baseOwner, base ] );
```

Comment:
There is `git.fetch` for this

Reply:
fixed in https://github.com/woocommerce/woocommerce/pull/38510/commits/bc38e72a64070f1cc95c37e492f8dcb06a1d9ede